chains/irisnet/conn/secret_connection.go

```diff
@@ -256,6 +256,7 @@ func (sc *SecretConnection) SetWriteDeadline(t time.Time) error {
 	return sc.conn.(net.Conn).SetWriteDeadline(t)
 }
 
+// genEphKeys generates Ephemeral key-pair
```

Comment:
This is a library imported from tendermint itself. No need to put commnts support here.